docs/index.bs

```diff
@@ -2780,14 +2797,16 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
       :: None
 
       1. If |serviceWorker| is not running, abort these steps.
```

Comment:
Link to your "running" definition?

Reply:
I wasn't sure what to do here. My definition doesn't work because it will abort the steps if the worker is starting up and hasn't started its event loop. On the other hand, the "running" used in the existing spec isn't defined. I wonder if we can just delete this step? The HTML equivalent doesn't have it:
https://html.spec.whatwg.org/multipage/workers.html#terminate-a-worker

Or else say something like, "If the steps of the "Create a separate parallel execution environment" step in the Run Service Worker algorithm are not being executed, abort these steps".  ?

Reply:
Deleting the step seems best I think